components/ServiceEdit/ServiceEdit.js

```diff
@@ -3,10 +3,36 @@ import s from './ServiceEdit.css'
 import icons from '../../icons/css/icons.css'
 import { gender } from '../../lib/eligibilities'
 import { age } from '../../lib/eligibilities'
+import { uuid } from '../../lib/uuid'
 import TimeRangePicker from '../TimeRangePicker'
 import ToggleButton from '../ToggleButton'
 
+const scheduleSorter = {
+  "-":        -1,
+  "Sunday":    0,
+  "Monday":    10000,
+  "Tuesday":   20000,
+  "Wednesday": 30000,
+  "Thursday":  40000,
+  "Friday":    50000,
+  "Saturday":  60000
+}
```

Comment:
So the existing times range 0-2359.  This was the easiest way I could think of to make sure that the previous day *always* weighted less than the next day.